rpm/gpsd.spec

```diff
@@ -113,15 +114,17 @@ ln -s ../gpsd.service %{buildroot}/lib/systemd/system/multi-user.target.wants/gp
 %postun
 # Don't restart the service
 %systemd_postun gpsd.service gpsd.socket
-
 %files
-/lib/systemd/system/gpsd.service
-/lib/systemd/system/multi-user.target.wants/gpsd.service
+%{_unitdir}/gpsd.service
+%{_unitdir}/multi-user.target.wants/gpsd.service
 %{_sbindir}/gpsd
 %{_sbindir}/gpsdctl
+%exclude %{_docdir}/*
+%exclude /share/gpsd/icons/gpsd-logo.png
 
 %files -n libgps
 %{_libdir}/libgps.so.*
+%{_libdir}/libgpsdpacket.so*
```

Comment:
In fedora spec this library is in python3 package https://src.fedoraproject.org/rpms/gpsd/blob/rawhide/f/gpsd.spec